Sources/MapboxNavigation/NavigationMapViewIdentifiers.swift

```diff
@@ -14,6 +14,7 @@ extension NavigationMapView {
         static let waypointCircleLayer = "\(identifier)_waypointCircleLayer"
         static let waypointSymbolLayer = "\(identifier)_waypointSymbolLayer"
         static let buildingExtrusionLayer = "\(identifier)buildingExtrusionLayer"
```

Comment:
While you’re here, there’s a missing `_` in the identifier. It isn’t causing any problems, but consistency might make refactoring easier down the road.